internal/pkg/vault/secrets_test.go

```diff
@@ -789,10 +789,10 @@ func getTestSecretsData() map[string]map[string]string {
 	}
 }
 
-func getTestSecretsKeysData() map[string]map[string][]string {
+func listTestSecretsKeysData() map[string]map[string][]string {
 	return map[string]map[string][]string{
 		"data": {
-			"keys": {"one", "uno", "two", "dos", "three", "tres"},
+			"keys": {"secret1", "secret2", "secret3/", "secret4/secret1"},
```

Comment:
As per https://www.vaultproject.io/api-docs/secret/kv/kv-v1, Vault will never actually return "secret4/secret1".  Remove this.